src/adapter.ts

```diff
@@ -85,9 +104,13 @@ export function toGuidelimeStep(classicWowStep: Array<string>): GuidelimeStep {
   };
 
   const getObjective = (): string => {
-    if (actionLine === 'Complete Objective') {
+    if (
+      ['Complete Objective', 'Complete Quest', 'Progress Objective', 'Progress Quest'].includes(
```

Comment:
I don't think we should have `Complete Quest` in this list, as Guidelime already has integration for showing the quest objectives in its step log, so we'll just see duplicate info.